composer/loggers/logger.py

```diff
@@ -199,6 +185,43 @@ def data_batch(self, data: Dict[str, Any]) -> None:
         self.data(LogLevel.BATCH, data)
 
 
+# Monkey patch Logger.file_artifact method docstring
```

Comment:
oh i see.. assume there is a reason for this.

Reply:
maybe put these at the start of the file, so that we don't need to moneky patch?